web/admin/src/App.js

```diff
@@ -246,6 +253,30 @@ export default class App extends Component {
     )
   }
 
+  formatDataForExport = () => {
+    let parsedData = []
+    const completed = this.state.attendees.filter(a => this.isDone(a.id))
+    this.state.attendees.forEach(attendee => {
+      if (this.state.allCodesByUser[attendee.id]) {
+        if (this.state.allCodesByUser[attendee.id].scans){
+          let gameWinningUser = completed.find(user => user.id === attendee.id)
+          const gameWinner = gameWinningUser ? "True" : null
+          let parsedUser = {First_Name: attendee.firstName, Last_Name: attendee.lastName, Email: attendee.email, Title: attendee.title, Company: attendee.company, Game_Winner: gameWinner}
+          let scans = Object.keys(this.state.allCodesByUser[attendee.id].scans)
+          scans.forEach(scan => {
```

Comment:
This seems to be creating duplicate CSV attendee rows, one for each scan.  Per DDDP-488, What we want is one row per attendee, with columns for:
1. Did they complete all the categories?
2. For each category:
   1. "Scans for 'CATEGORY_NAME'" (number)
   2. "Completed 'CATEGORY_NAME'" (boolean)

Note that this means that every attendee row will have each column filled in.